src/Sentry.NLog/ConfigurationExtensions.cs

```diff
@@ -0,0 +1,142 @@
+        /// <param name="optionsConfig">An optional action for configuring the Sentry target options.</param>
+        /// <returns>The configuration.</returns>
+        public static LoggingConfiguration AddSentry(this LoggingConfiguration configuration,
+                                                           Action<SentryNLogOptions> optionsConfig = null)
+        {
+            return configuration.AddSentry(null, DefaultTargetName, optionsConfig);
+        }
+
+        /// <summary>
+        /// Adds a target for Sentry to the NLog configuration.
+        /// </summary>
+        /// <param name="configuration">The NLog configuration.</param>
+        /// <param name="dsn">          
+        /// The sentry DSN. If DSN is not set, the SDK will look for an environment variable called SENTRY_DSN.
+        /// If nothing is found, SDK is disabled.
+        /// </param>
+        /// <param name="optionsConfig">An optional action for configuring the Sentry target options.</param>
+        /// <returns>The configuration.</returns>
+        public static LoggingConfiguration AddSentry(this LoggingConfiguration configuration,
+                                                           string dsn,
+                                                           Action<SentryNLogOptions> optionsConfig = null)
+        {
+            return configuration.AddSentry(dsn, DefaultTargetName, optionsConfig);
+        }
+
+        /// <summary>
+        /// Adds a target for Sentry to the NLog configuration.
+        /// </summary>
+        /// <param name="configuration">The NLog configuration.</param>
+        /// <param name="dsn">          The sentry DSN.</param>
+        /// <param name="targetName">   The name to give the new target.</param>
+        /// <param name="optionsConfig">An optional action for configuring the Sentry target options.</param>
+        /// <returns>The configuration.</returns>
+        public static LoggingConfiguration AddSentry(this LoggingConfiguration configuration,
+                                                           string dsn,
+                                                           string targetName,
+                                                           Action<SentryNLogOptions> optionsConfig = null)
+        {
+            var options = new SentryNLogOptions();
+
+            optionsConfig?.Invoke(options);
+
+            return configuration.AddSentryTargetInternal(dsn, targetName, options);
+        }
+
+        /// <summary>
+        /// Adds a target for Sentry to the NLog configuration, with more configuration available.
+        /// </summary>
+        /// <param name="configuration">The NLog configuration.</param>
+        /// <param name="targetConfig"> An optional configuration for the Sentry target.</param>
+        /// <returns>The configuration.</returns>
+        public static LoggingConfiguration AddSentryTarget(this LoggingConfiguration configuration, Action<SentryTarget> targetConfig)
```

Comment:
Yeah, basically the only difference was that one had an `Action<SentryTarget>` and one used an `Action<SentryNLogOptions>`, because there were some options that were only configured on the `Target` class, but now I think if someone wants to do that they can just get a reference to the target the normal way. I'll clean that up.